src/index.ts

```diff
@@ -96,8 +100,13 @@ export class WaxJS {
     ) => void;
     metricURL?: string;
     returnTempAccounts?: boolean;
+    chainName?: string;
   }) {
-    this.rpc = new JsonRpc(rpcEndpoint);
+    this.rpc = this.registryRpc = new JsonRpc(rpcEndpoint);
```

Comment:
On second thought I think we should have a registryEndpoint argument in the constructor params that defaults to null, and here we should have:
```
this.rpc = new JsonRpc(rpcEndpoint);
this.registryRpc = new JsonRpc(registryEndpoint || rpcEndpoint);
```